app/src/main/java/net/gsantner/opoc/util/ShareUtil.java

```diff
@@ -1093,10 +1093,10 @@ private Activity greedyGetActivity(final Activity... activity) {
      * Requires storage access framework permission for external storage (SD)
      *
      * @param file  The file object (file/folder)
-     * @param isDir Wether or not the given file parameter is a directory
      * @return Wether or not the file can be written
      */
-    public boolean canWriteFile(final File file, final boolean isDir) {
+    public boolean canWriteFile(final File file) {
+        final boolean isDir = file.isDirectory();
```

Comment:
I think this change is ok

Reply:
hmm, it can produce wrong result. when previously isDir=true is passed, then it would be checked if the parentdirectory is writable.   
Generally this function has been written quite some time ago, might need some improvements/fixes - hope adding this not breaks additionally

Reply:
No, you're right. I will revert this change. No need to possibly introduce additional bugs if it isn't needed.